src/app/common/components/ValidatedPasswordInput.tsx

```diff
@@ -7,7 +7,8 @@ import {
   TextInput,
   TextInputProps,
 } from '@patternfly/react-core';
-import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+import EyeIcon from '@patternfly/react-icons/dist/esm/icons/eye-icon';
+import EyeSlashIcon from '@patternfly/react-icons';
```

Comment:
I think this one can be just like the others, maybe the linter bit ya?

`import EyeSlashIcon from '@patternfly/react-icons/dist/esm/icons/eye-slash-icon';`

Reply:
I left that one aside because it initially didn't work for me but I must have done a typo because it's working now.
Thanks for insisting! ;)